src/main/java/com/jprobstats/stats/descriptive/Maximum.java

```diff
@@ -0,0 +1,16 @@
+package com.jprobstats.stats.descriptive;
+
+public class Maximum implements UnivariateStatistic {
+
+    @Override
+    public double evaluate(double[] values) throws Exception {
```

Comment:
let the argument be List<Number> values instead of double[] ,please change  it in every other class and places too.I thick its best to have a data of Arraylist of Number rather then a double values . What do u say...